### ISSUE TITLE
'dict' object has no attribute 'has_key'

### ISSUE THREAD

**Issue by felixhao28** [colinta/SublimeStringEncode#30]
```
Traceback (most recent call last):
  File "C:\Program Files\Sublime Text 3\sublime_plugin.py", line 798, in run_
    return self.run(edit)
  File "C:\Users\XXX\AppData\Roaming\Sublime Text 3\Packages\SublimeStringEncode\string_encode.py", line 99, in run
    elif kwargs.has_key('source'):
AttributeError: 'dict' object has no attribute 'has_key'
```

In python3 [has_key is deprecated](https://docs.python.org/3.1/whatsnew/3.0.html#builtins).

**Comment by colinta** [comment]
aww shoot.  one sec.

thanks for reporting!

**Comment by colinta** [comment]
Fixed in 2.2.1.
